src/libraries/System.CodeDom/src/Microsoft/VisualBasic/VBCodeGenerator.cs

```diff
@@ -865,7 +865,11 @@ protected override void GenerateArrayCreateExpression(CodeArrayCreateExpression
                 string typeName = GetTypeOutput(e.CreateType);
                 Output.Write(typeName);
 
-                if (typeName.IndexOf('(') == -1) // string.Contains(char) is .NetCore2.1+ specific
+#if NETCOREAPP
+                if (typeName.Contains('('))
```

Comment:
Looks this is causing the[ VB test failures](https://github.com/dotnet/runtime/issues/53522) as @ericstj [ mentioned here](https://github.com/dotnet/runtime/issues/53522#issuecomment-858157547)
```suggestion
                if (!typeName.Contains('('))
```